Description: Fix FTBFS with glibc 2.28
Author: Adrian Bunk <bunk@debian.org>
Bug-Debian: https://bugs.debian.org/917055

This patch is taken from
http://ftp.debian.org/debian/pool/main/b/blktool/blktool_4-7.1.debian.tar.xz

Upstream-Status: Inappropriate [upstream is dead]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
--- blktool-4.orig/blktool.c
+++ blktool-4/blktool.c
@@ -18,6 +18,7 @@
 #include <unistd.h>
 #include <sys/stat.h>
 #include <sys/ioctl.h>
+#include <sys/sysmacros.h>
 #include <fcntl.h>
 #include <errno.h>
 #include <getopt.h>
